Packs/JoeSecurity/Integrations/JoeSecurityV2/JoeSecurityV2.yml

```
@@ -0,0 +1,366 @@
+category: Forensics & Malware Analysis
+commonfields:
+  id: JoeSecurityV2
+  version: -1
+configuration:
+- name: url
+  display: Your server URL
+  required: true
+  defaultvalue: https://jbxcloud.joesecurity.org/api
+  type: 0
+- name: credentials
+  displaypassword: API Key
+  type: 9
+  required: true
+  hiddenusername: true
+- name: insecure
+  display: Trust any certificate (not secure)
+  required: false
+  type: 8
+  additionalinfo:
+- name: proxy
+  display: Use system proxy settings
+  required: false
+  type: 8
+  additionalinfo:
+- name: create_relationships
+  display: Create relationships
+  required: false
+  type: 8
+- name: Reliability
+  additionalinfo: Reliability of the source providing the intelligence data.
+  defaultvalue: C - Fairly reliable
+  display: Source Reliability
+  options:
+  - A+ - 3rd party enrichment
+  - A - Completely reliable
+  - B - Usually reliable
+  - C - Fairly reliable
+  - D - Not usually reliable
+  - E - Unreliable
+  - F - Reliability cannot be judged
+  required: true
+  type: 15
+description: Access the full set of possibilities the JoeSandbox Cloud provides via RESTful Web API v2.
+display: Joe Security v2
+name: JoeSecurityV2
+script:
+  commands:
+  - name: joe-is-online
+    description: Check if the Joe Sandbox analysis back end is online or in maintenance mode.
+    deprecated: false
+    arguments: []
+    outputs: []
+  - name: joe-list-analysis
+    description: List all analyses in descending order, starting with the most recent. Pagination is available by supplying page and page size arguments or by limit. (default 50)
```

```suggestion
    description: Lists all analyses.
```
pagination information is implied from the arguments. relevant for other commands.

---

general note to ease the doc review process: We write descriptions in present simple, for example in the above command, `Lists` and not `List`.